src/core/index.js

```diff
@@ -119,12 +119,12 @@ if (instanceNamespaces) {
         componentCreators,
         lifecycle,
         componentRegistry,
-        getImmediatelyAvailableTools(componentAbbreviation) {
+        getImmediatelyAvailableTools(componentNamespace) {
           return {
             config,
             optIn,
             eventManager,
-            logger: logController.createComponentLogger(componentAbbreviation)
+            logger: logController.createComponentLogger(componentNamespace)
```

Comment:
This appears to have been a bug. We should have been using the component's namespace in the logger previously.